aether-kernel/aether/kernel/api/views.py

```diff
@@ -310,6 +312,16 @@ class SubmissionViewSet(CustomViewSet):
     serializer_class = serializers.SubmissionSerializer
     filter_class = filters.SubmissionFilter
 
+    def get_queryset(self):
+        queryset = models.Submission.objects.all()
```

Comment:
This line will remove previous filters executed by the `SubmissionFilter` class like the `project` filter :scream: 
The correct line is `queryset = self.queryset` :stuck_out_tongue:

Reply:
Oh thanks, I thought this ran before the other filters. I'll take a look.

Reply:
Maybe you are right, but imagine that the default `queryset` is something like this:

```python
    queryset = Project.objects \
                      .prefetch_related('xforms', 'xforms__media_files') \
                      .order_by('name')
```
or
```python
    queryset = models.Project \
                     .objects \
                     .values('id', 'name', 'created') \
                     .annotate(
                         first_submission=Min('submissions__created'),
                         last_submission=Max('submissions__created'),
                         submissions_count=Count('submissions__id', distinct=True),
                         entities_count=Count('submissions__entities__id', distinct=True),
                     )
```

:woman_shrugging:

Reply:
Yes, agreed it's much clearer to use `self.queryset`. Fixing.